pacemaker/pacemaker.go

```diff
@@ -69,8 +70,10 @@ type RoundRobin struct {
 	schedule   []hotstuff.ReplicaID
 	timeout    time.Duration
 
-	resetTimer  chan struct{} // sending on this channel will reset the timer
-	stopTimeout func()        // stops the new-view interrupts
+	resetTimer           chan struct{}   // sending on this channel will reset the timer
+	stopTimeout          func()          // stops the new-view interrupts
+	timeoutContext       context.Context // timeoutContext is passed down to the RPC's. A new context is created for every transmission.
```

Comment:
If a new context is created for every transmission, couldn't it be a local variable instead?